### ISSUE TITLE
Lolipop issue (4.4.4)

### ISSUE THREAD

**Issue by raviteja786143** [rey5137/material#197]
Exception: android.view.InflateException: Binary XML file line #32: Error inflating class com.rey.material.widget.Button

**Comment by Gaudon** [comment]
I have the same problem on my pre-lollipop devices. I mentioned it last week in #189 but haven't received a response yet.

**Comment by Gaudon** [comment]
Still no update on this? @rey5137 any comments here?

**Comment by Gaudon** [comment]
Are you using multidex by chance?

**Comment by Gaudon** [comment]
I see a bunch of things like this now too:

DexOpt: unable to optimize static field ref 0x4e54 at 0x30 in Lcom/rey/material/util/ViewUtil;.applyStyle
DexOpt: unable to optimize static field ref 0x4e55 at 0x42 in Lcom/rey/material/util/ViewUtil;.applyStyle
DexOpt: unable to optimize static field ref 0x4e56 at 0x5e in Lcom/rey/material/util/ViewUtil;.applyStyle
DexOpt: unable to optimize static field ref 0x4e57 at 0xb6 in Lcom/rey/material/util/ViewUtil;.applyStyle

Not sure if it's proguard, multidex or something else related.. but I can't get this library to work on pre 5.0 devices anymore.

**Comment by mobilestar2015** [comment]
I experienced same issue when using com.rey.material.widget.Button and find out the cause of the problem.

Here's my wrong implementation for inflating layout which contains com.rey.material.widget.Button.
View v = LayoutInflater.from((getApplicationContext()).inflate(R.id.drawer_layout);

Then I changed the code as followings.
LayoutInflater inflater = (LayoutInflater)getSystemService(LAYOUT_INFLATER_SERVICE);
View v = inflater.inflate(R.layout.row_drawer, null);

It works for me. I wonder if your case is same with my case but hope it would help anybody experiencing this issue.
